src/components/controls/FilterListPanel.js

```diff
@@ -59,11 +59,13 @@ function FilterListPanel({
           onClickCheckbox={() => onSelectFilter(key, matchingKeys)}
           color={assignedColor}
         />
-        {Object.keys(children).length > 0
-          ? Object.entries(children).map((filter) =>
+        {Object.keys(children).length > 0 ? (
+          <ul>
```

Comment:
`li > li` is not valid, but `li > ul > li` is. I didn't adjust the styles because they already seem to have styling for `ul` (from the parent one).